### PR TITLE
Remove unrequired yamllint statements & use local action

### PR DESCRIPTION
**Pull request by paddyroddy**
The project board action wasn't using the action locally - which is in keeping with the others

### CODE REVIEW

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
   pull_request:
-    types:
-      - opened
-      - ready_for_review
-      - reopened
-      - synchronize
 
 jobs:
   linting:
-    if: github.event.pull_request.draft == false
```
Comment (paddyroddy) [inline .github/workflows/linting.yml:19]: Removing for simplicity and the repo is public anyway

File: .github/workflows/manage-projects.yml [hunk 2]
```diff
@@ -11,8 +10,10 @@ jobs:
   add-issue-to-project:
     runs-on: ubuntu-latest
     steps:
-      # yamllint disable-line rule:line-length
-      - uses: UCL-MIRSG/.github/actions/add-to-project@610c6ef540b030d6346ad1ba0fd5eb270efd7526 # v0.95.0
+      - name: Checkout source
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+
+      - uses: ./actions/add-to-project
```
Comment (paddyroddy) [inline .github/workflows/manage-projects.yml:16]: Will always use `main`

File: .github/workflows/linting.yml [hunk 3]
```diff
@@ -1,28 +1,20 @@
...
   push:
     branches:
       - main
       - renovate/**
   pull_request:
-    types:
-      - opened
-      - ready_for_review
-      - reopened
-      - synchronize
```
Comment (p-j-smith) [inline .github/workflows/linting.yml:15]: was there an issue with linting not running when it should?

File: .github/workflows/linting.yml [hunk 4]
```diff
@@ -1,28 +1,20 @@
...
 
 jobs:
   linting:
-    if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     steps:
       - name: Checkout source
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
 
-      - name: Run pre-commit over the repo
```
Comment (p-j-smith) [inline .github/workflows/linting.yml:25]: just wondering why you removed the name?

**Review by p-j-smith — APPROVED** [review]
looks good, couple of questions but nothing blocking

File: .github/workflows/linting.yml (hunk 3 above)
Comment (paddyroddy) [reply .github/workflows/linting.yml:15]: This was thinking about reducing CI usage after MIRSG-locked-out-of-GitHub-Actions gate, but this is a private repo, so not worth it

File: .github/workflows/linting.yml (hunk 4 above)
Comment (paddyroddy) [reply .github/workflows/linting.yml:25]: Purely consistency, the action name should be verbose enough!

**Pull request merged**